openassessment/templates/openassessmentblock/oa_rubric.html

```diff
@@ -64,7 +64,7 @@ <h4 class="question__title ui-toggle-visibility__control">
                 <textarea
                     id="assessment__rubric__question--feedback__value"
                     placeholder="{{ rubric_feedback_default_text }}"
-                    maxlength="500"
+                    maxlength="100000"
```

Comment:
Obviously 300 was too small but I'm wondering if 100,000 is too large. Who's going to read a comment or review that large? Are you strongly wedded to this value or could we reduce it?